app-text/pad/pad-1.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A freeware notepad program, similar to Apple's Notepad app"
+DESCRIPTION="Pad is a freeware notepad program. Anyone who has used Apple's \
+Notepad app (included with the system software) should find it familiar.\
+ It supports multiple pages, the standard text editing commands \
+(Cut, Copy, Paste), as well as Drag 'n' Drop editing"
+HOMEPAGE="https://github.com/HaikuArchives/Pad"
+COPYRIGHT="1997 Jesse Hall"
+LICENSE="Pad"
+REVISION="1"
+SOURCE_URI="git://github.com/HaikuArchives/Pad"
```

Comment:
Some old recipes still use `git:` for their `SOURCE_URI` but we are now using https:
```diff
 REVISION="1"
-SOURCE_URI="git://github.com/HaikuArchives/Pad"
+srcGitRev="26748fcb828358394d017d178ebe3a73413961a0"
+SOURCE_URI="https://github.com/HaikuArchives/Pad/archive/$srcGitRev.tar.gz"
-SOURCE_DIR=""
-CHECKSUM_SHA256=""
+CHECKSUM_SHA256="0d3d3d3fa9d00258ca82c06bbf5ef1e6f6fc78d0306b82faeaacb6dea82ffb8f"
+SOURCE_FILENAME="Pad-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Pad-$srcGitRev"
-#PATCHES=""
+
-ARCHITECTURES="x86 x86_gcc2 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
 PROVIDES="
```